wagtailio/project_styleguide/templates/patterns/components/nav-cta/nav-cta.html

```diff
@@ -0,0 +1,17 @@
+<a class="nav-cta" href="{{ cta.url }}" title="{{ cta.text }}">
```

Comment:
question: what's the title attribute needed for?

Reply:
removed :)